config/initializers/assets.rb

```diff
@@ -9,4 +9,6 @@
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
 Rails.application.config.assets.precompile += %w(application_public.js)
+Rails.application.config.assets.precompile += %w( custom.css )
```

Comment:
I think you should move custom.css to the line above (`%w(a b)` creates an array `[a, b]`, it's exactly what this is for)